ExRam.Gremlinq.Providers.WebSocket/IWebSocketConfigurationBuilder.cs

```diff
@@ -12,6 +13,8 @@ public interface IWebSocketConfigurationBuilder : IGremlinQueryEnvironmentBuilde
 
         IWebSocketConfigurationBuilder SetAlias(string alias);
 
+        IWebSocketConfigurationBuilder SetConnectionPoolSettings(ConnectionPoolSettings connectionPoolSettings);
```

Comment:
Can you make the parameter an Action<ConnectionPoolSettings> so the configuration may take place on an existing, maybe already configured instance. That way, multiple configurations can add up.

Reply:
And maybe even rename it "ConfigureConnectionPool". Set...Settings seems a bit weird.

Reply:
I created Func<ConnectionPoolSettings>